README.md

```diff
@@ -9,16 +9,16 @@ Instead of manually calling `use()` for every request, `use()` is called automat
 ## Example
 
 ```js
+var superagent = require('superagent'),
+    superagentUse = require('superagent-use'),
+    prefix = require('superagent-prefix'); /* A sample superagent plugin/middleware. */
 
-var superagent = require('superagent');
-/* Pass the superagent object to superagent-use
-   in order to be provided with new `use` functionality */
-require('superagent-use')(superagent);
-var prefix = require('superagent-prefix')('https://api.example.com');
+/* superagentUse returns a clone of the superagent provided with the new functionality. */
+var request = superagentUse(superagent);
```

Comment:
I'd disagree because the SuperAgent uses this name in the docs and I guess that as a result this name is even more popular. If you still prefer `superagent` over `request` we can use it tho. (?)